renovate.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", ":prHourlyLimit4", ":semanticCommitTypeAll(chore)"],
+  "extends": ["config:recommended", ":prHourlyLimit4", ":semanticCommitTypeAll(chore)"],
```

Comment:
Oh, I did not know about it 😅
Thanks to you I know one more thing about renovate 💚